src/components/HomePage.jsx

```diff
@@ -0,0 +1,121 @@
+import './Styles/HomePage.css';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Countries from './Countries';
+import { getCountriesData } from '../Redux/Countries/CountriesSlice';
+import world from '../images/World.svg';
+import Africa from '../images/Africa.svg';
+import Americas from '../images/Americas.svg';
+import Asia from '../images/Asia.svg';
+import Europe from '../images/Europe.svg';
+import Oceania from '../images/Oceania.svg';
```

Comment:
- [ ] As per the `requirement`, there should not be any static code in your `capstone` project, your data of any sort or type should come from an **API**, and using the redux store you can use that data to display in your project. Kindly try to find a way to get those continent `map` images from an *API* not statically put them in this `file`.